src/pages/StableSwapPoolRemoveLiquidity/StableSwapRemoveLiquidityCurrencyRow.tsx

```diff
@@ -11,13 +11,14 @@ import _ from 'lodash'
 type Props = {
   currency: Currency
   value: string
+  index: number
 }
 
-export default function StableSwapRemoveCurrencyRow({ currency, value }: Props) {
+export default function StableSwapRemoveCurrencyRow({ currency, value, index }: Props) {
   const { t } = useTranslation()
 
   return (
-    <InputPanel>
+    <InputPanel id={`remove-liquidity-input-token${['a', 'b', 'c'][index]}`}>
```

Comment:
nit I'd changed the field names from `a/b/c` to reflect their index positions on the solidity contract -- if it's easier you can do `remove-liquidity-input-token[index]`

ndb tho this is working fine, too

Reply:
will keep it if that's cool just cause it's the same as the default amm remove liquidity too

they're just ids anyways so it's just this